pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -178,7 +178,7 @@ func isIssuedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 
 // isOlderThan checks that t is a non-zero time after time.Now() + d.
```

Comment:
pre-existing, but the doc was backwards, which really confused me reviewing this... it should be `isOlderThan checks that t is a non-zero time before time.Now() - d`

Reply:
this comment is still to be addressed